browser/brave_content_browser_client.cc

```diff
@@ -334,6 +382,12 @@ void BraveContentBrowserClient::RegisterBrowserInterfaceBindersForFrame(
     map->Add<brave_search::mojom::BraveSearchDefault>(
         base::BindRepeating(&BindBraveSearchDefaultHost));
   }
+
+#if BUILDFLAG(BRAVE_ADS_ENABLED)
```

Comment:
I don't think we need or want a buildflag for this and I don't see where the default value is set anyway

Reply:
Yes, seems like this buildflag has no effect in this code. I will remove it.
Also I think it is needed to remove `if (brave_ads_enabled)` checks from .gn files

Reply:
Fixed, buildflag was removed here: https://github.com/brave/brave-core/pull/9769